crates/bindings-csharp/BSATN.Runtime/BSATN.Runtime.csproj

```diff
@@ -26,13 +26,15 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <ProjectReference Include="../BSATN.Codegen/BSATN.Codegen.csproj" OutputItemType="Analyzer" PrivateAssets="all" />
+    <ProjectReference Include="../BSATN.Codegen/BSATN.Codegen.csproj" PrivateAssets="all" OutputItemType="analyzer" ReferenceOutputAssembly="false" />
   </ItemGroup>
 
   <ItemGroup>
     <None Include="../logo.png" Pack="true" PackagePath="" />
     <None Include="../Runtime/README.md" Pack="true" PackagePath="" />
     <None Include="../LICENSE" Pack="true" PackagePath="" />
+    <!-- We want all users who depends on BSATN.Runtime to automatically get the Roslyn codegen component as well. -->
+    <None Include="../BSATN.Codegen/bin/$(Configuration)/netstandard2.0/SpacetimeDB.BSATN.Codegen.dll" Pack="true" PackagePath="analyzers/dotnet/cs" />
```

Comment:
Dumb question - this says `Pack="true"` but the project above says `<IsPackable>false</IsPackable>` - I assume those don't actually conflict despite kinda sounding like they would?

Reply:
another one - does this require any `IncludeRuntimeDependency="false"` like `Codegen.csproj` does below?

Reply:
> Dumb question - this says `Pack="true"` but the project above says `<IsPackable>false</IsPackable>` - I assume those don't actually conflict despite kinda sounding like they would?

Yeah the idea here is that BSATN.Codegen should not be packable (aka publishable) as a separate NuGet package after this change, _but_ we explicitly pack the DLL it produces in the BSATN.Runtime package instead.

Reply:
Perfect, thanks!